### ISSUE TITLE
Extra output from primitive breaks primitive description

### ISSUE THREAD

**Issue by mitar** [uncharted-distil/distil-primitives#34]
See [this log](https://storage.googleapis.com/gitlab-gprd-artifacts/f4/bc/f4bc1cf5dcb555ebacfa12ea66d4dc0cb7154a2f9cb17729561e410fcaa741e2/2019_06_17/232369317/240977575/job.log?response-content-type=text%2Fplain%3B%20charset%3Dutf-8&response-content-disposition=inline&GoogleAccessId=gitlab-object-storage-prd@gitlab-production.iam.gserviceaccount.com&Signature=W9b067PuvS1FgBea61CbMWQpo8fXD2CFeTmTtyjGwqbcvzHcx4jfR4NS4San%0AnPUyE7nUG9qgZj5ZKR2SpA%2B4uc3ys2wl3TGk8d0GxdFDcpmWE2wZ%2BPn6tEKT%0AJz03kaEGLstYRnH%2B2m3nKCwaCl9%2FXsCry%2BHL06KHQRTlKTRyxK3nI7opUoWm%0Aczge8XS%2BY7BqAZLSQPrNPVPEIO0WDC7WnEhKZ01eRuIHbRTEYyXT75twt%2Bp5%0AC4NNkU6KldEyUkkmiB0g1ZwWGX8ufHmTjRdaBh4UX7U41ObFBTGZbPOmUd7v%0AX0XieDOY0VzfFkSahfANptysT1lwH%2Fxfqr97IDwfBA%3D%3D&Expires=1560790284).

Primitive annotation has at the beginning:

```
Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex.
```

Not a good thing because it is not JSON compatible.

**Comment by mitar** [comment]
I think this should be fixed using the version which has this commit in: https://github.com/huggingface/pytorch-pretrained-BERT/commit/5c85fc3977c4dcad526233a4706cffcd89e59eb0

It uses logger instead of print, which would then not dirty the output.

It looks like it is released in `v0.6.2` version.

**Comment by mitar** [comment]
You have currently dependency on the 0.4.0 version.

**Comment by cdbethune** [comment]
0.6.2 had breaking interface changes, so I forked and fixed off of the 0.4.0 tag.
